cmd/repo-updater/repos/github_test.go

```diff
@@ -125,6 +125,35 @@ func TestGithubSource_GetRepo(t *testing.T) {
 	}
 }
 
+func TestGithubSource_RegOrg(t *testing.T) {
+	testCases := []struct {
+		str   string
+		match string
+		size  int
+	}{
+		{str: "org:gorilla", match: "gorilla", size: 2},
+		{str: "org:golang-migrate", match: "golang-migrate", size: 2},
+		{str: "sourcegraph", size: 0},
+		{str: "org:-sourcegraph", size: 0},
```

Comment:
How about testing some more edge cases that are covered in your regex? 

```suggestion
		{str: "org:-sourcegraph", size: 0},
		{str: "org:source graph", size: 0},
		{str: "org:thisorgnameisfartoolongtomatchthisregexp", size: 0},
		{str: "org:source org:graph", size: 0},
		{str: "org: sourcegraph", size: 0},
```